cts/cts-cli.in

```diff
@@ -274,9 +274,10 @@ def sanitize_output(s):
         (r'request=".*(crm_?[a-zA-Z0-9]+) ', r'request="\1 '),
         (r'request=".*iso8601', r'request="iso8601'),
         (r' start="[0-9][-+: 0-9]*Z*"', r' start=""'),
-        (r'/tmp/cts-cli\.[^/]*/shadow.cts-cli', r'/tmp/cts-cli.shadow/shadow.cts-cli'),
-        (r'^/tmp/cts-cli\.xmllint\.[^:]*:', r'/tmp/cts-cli.xmllint:'),
-        (r'^/tmp/cts-cli\.xmllint\.[^ ]* fails to validate', r'/tmp/cts-cli.xmllint fails to validate'),
+        (r'^/.*/cts-cli\.[^/]*/shadow.cts-cli', r'cts-cli.shadow/shadow.cts-cli'),
```

Comment:
This works right now, but if a test that's added later has the path appear somewhere besides the beginning of the line, this won't catch it. I appreciate that you also added a line to match on a space character; that would mitigate the issue. But it doesn't resolve it. Consider a double quote character before the slash, for example.

We might be able to match on word boundary. I can't remember if that's possible since `'/'` isn't a word character -- that might not matter though.

A better approach, for this and the other substitutions, might be to replace `/tmp` with `$TMPDIR` (using appropriate Python syntax) in the source strings, and keep the target strings the way you have them.

Reply:
I was looking at 2.1, where this is still a shell script that uses `$TMPDIR` as such. However, see https://github.com/ClusterLabs/pacemaker/pull/3822#discussion_r1945368270. We can always use `os.environ`.